tests/quality_pragmaitc/test_refinement.py

```diff
@@ -408,10 +405,11 @@ def shouldThereBeMoreThanOneApplicableQCreturnTheStricterOne():
     assert qc in goal.getInterpretation().getQualityConstraints(fullContext)
 
     fullContext.append(anotherContext)
-    assert stricter in goal.getInterpretation().getQualityConstraints(fullContext)
+    assert stricter in \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4895731)